cmd/get/metal3/baremetalhost.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 func getBareMetalHosts(currentContextPath string, namespace string, resourceName string, allNamespacesFlag bool, outputFlag string, showLabels bool, jsonPathTemplate string, allResources bool) bool {
-	_headers := []string{"namespace", "name", "age"}
+	_headers := []string{"namespace", "name", "state", "consumer", "online", "error", "age"}
```

Comment:
With this modification we will have only the headers populated, but not the fields; so this PR is incomplete.